### DISCUSSION TITLE
你好 我已成功配置在docker上了 我想请问一下 数据库改如何备份呢 位置在哪里呢

### DISCUSSION THREAD

**Discussion by 972513547** [swuecho/chat#374] [Q&A]
RT

**Comment by swuecho** [comment]
docker-compose 里面

```
 ports:
      - "5432:5432"
```

已经把port forward到了host

所以你用类似:

`psql postgres://username:password@localhost:port/dbname `

就直接可以连了. 

备份 pg_dump.

只是data 在 docker 的 filesystem 里面.

**Comment by swuecho** [comment]
https://github.com/swuecho/chat/discussions/267
